js-foundations.md

```diff
@@ -3,25 +3,35 @@
 ### Without using Google answer the following:
 
 1. List the data types in Javascript. Indicate which are primitives.
+2. strings 
+3. boolen
 
 2. What's the difference between =, ==, and === in JavaScript?
+=== is comparing a specific value, = compares one value 
 
```

Comment:
The single = assigns a value, the == evaluates equality of two things, while the === is a strict comparison evaluating the equality as well as the datatype